crud/select/plan.lua

```diff
@@ -208,7 +208,7 @@ function select_plan.new(space, conditions, opts)
 
     -- get sharding key value
     local sharding_key
-    if scan_value ~= nil and (scan_iter == box.index.EQ or scan_iter == box.index.REQ) then
+    if scan_value ~= nil and scan_after_tuple == nil and (scan_iter == box.index.EQ or scan_iter == box.index.REQ) then
```

Comment:
Why?

It seems only first query could be executed on single shard if full sharding key is specified.